corehq/apps/smsbillables/models.py

```diff
@@ -26,11 +26,13 @@
 from corehq.apps.smsbillables.utils import (
     get_twilio_message,
     get_infobip_message,
+    get_pinpoint_message,
     log_smsbillables_error,
 )
 from corehq.messaging.smsbackends.test.models import SQLTestSMSBackend
 from corehq.messaging.smsbackends.twilio.models import SQLTwilioBackend
 from corehq.messaging.smsbackends.infobip.models import InfobipBackend
+from corehq.messaging.smsbackends.amazon_pinpoint.models import PinpointBackend
```

Comment:
Until fairly recently this module did not reference specific gateway implementations. I know you did not add the `SQLTestSMSBackend` or `SQLTwilioBackend` references, but it would be best not to add more. Instead, the messaging framework should delegate to the backend instance when backend-specific logic is required.